### PR TITLE
Add html support

### LINKED ISSUE #9: Add suport for rich-text import
## The problem to solve

If you create a column in your CSV file for a field of type `area`, it is imported as plaintext. HTML is not supported. Any special characters like `< `and `>` should be escaped so the user can see them

## Proposed solution

To import areas as rich text HTML markup, there should be a property of the area field in your schema (for example `importAsRichText: true`)

### PR DESCRIPTION
**Pull request by justyna13**
## Summary

 If it closes an existing Github issue, include "Closes #9"*

## What kind of change does this PR introduce?
- New feature

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -181,4 +188,23 @@ describe('Pieces Importer', function () {
     assert(processedErr);
     assert(processedErr === 'You can have only one key column for updates.');
   });
+
+  it('If piece\'s schema has field with importAsRichText: true option, should convert html content to richText area', async () => {
```
Comment (ETLaurent) [inline test/test.js:192]:
```suggestion
  it('should convert html content to richText area if piece\'s schema has field with importAsRichText: true option', async () => {
```

**Review by ETLaurent — COMMENTED** [review]
LGTM, tested it ✅ 
@boutell you might want to take a look, in case I'm not seeing something that could be problematic.

Thanks @justyna13!
Could you add the following section in the readme?

```md
## Importing rich text (HTML) rather than plaintext

By default, if you create a column in your CSV file for a field of type `area`, it will be imported as plaintext. Any special characters like `<` and `>` will be escaped so the user can see them. HTML is not supported.

To import areas as rich text HTML markup, add the `importAsRichText: true` option to
the `area` field in your schema.
```

**Review by boutell — APPROVED** [review]

**Comment by justyna13** [comment]
@ETLaurent Readme updated

**Review by ETLaurent — APPROVED** [review]

**Comment by ETLaurent** [comment]
Thanks @justyna13!

**Pull request merged**